src/edu/asu/plp/tool/prototype/Main.java

```diff
@@ -413,45 +463,843 @@ private ProjectExplorerTree createProjectTree()
+		itemVGA.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemPLPID = new MenuItem("PLPID");
+		itemPLPID.setGraphic(new ImageView(new Image("toolbar_sim_plpid.png")));
+		itemPLPID.setAccelerator(
+				new KeyCodeCombination(KeyCode.NUMPAD6, KeyCombination.CONTROL_DOWN));
+		itemPLPID.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemGPIO = new MenuItem("GPIO");
+		itemGPIO.setGraphic(new ImageView(new Image("toolbar_sim_gpio.png")));
+		itemGPIO.setAccelerator(
+				new KeyCodeCombination(KeyCode.NUMPAD7, KeyCombination.CONTROL_DOWN));
+		itemGPIO.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemExitSim = new MenuItem("ExitSimulation");
+		itemExitSim.setAccelerator(new KeyCodeCombination(KeyCode.F11));
+		itemExitSim.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		cyclesSteps.getItems().addAll(itemOne, itemFive, itemTwenty, itemHundred,
+				itemFiveThousand);
+		views.getItems().addAll(itemCpuView, itemCpuWindow, itemSimControlWindow);
+		toolsSubMenu.getItems().addAll(itemioRegistry, itemASMView,
+				new SeparatorMenuItem(), itemCreateMemVis, itemRemoveMemVis,
+				itemDisplayBus);
+		ioDevices.getItems().addAll(itemLedArray, itemSwitches, itemSevenSeg, itemUART,
+				itemVGA, itemPLPID, itemGPIO);
+		simulation.getItems().addAll(itemStep, itemReset, new SeparatorMenuItem(),
+				itemRun, cyclesSteps, itemClearBreakpoints, new SeparatorMenuItem(),
+				views, toolsSubMenu, ioDevices, new SeparatorMenuItem(), itemExitSim);
+				
+		// Menu Items Under "Help"
+		Menu help = new Menu("Help");
+		MenuItem itemQuickRef = new MenuItem("Quick Reference");
+		itemQuickRef.setAccelerator(new KeyCodeCombination(KeyCode.F1));
+		itemQuickRef.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemOnlineManual = new MenuItem("Online Manual");
+		itemOnlineManual.setOnAction((event) -> {
+			onlineManualWeb();
+		});
+		MenuItem itemReportIssue = new MenuItem("Report Issue (Requires Google Account");
+		itemReportIssue.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemGoogleIssues = new MenuItem("Open Google Code Issues Page");
+		itemGoogleIssues.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemAboutPLP = new MenuItem("About PLP Tool...");
+		itemAboutPLP.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemSWLicense = new MenuItem("Third Party Software License");
+		itemSWLicense.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		help.getItems().addAll(itemQuickRef, itemOnlineManual, new SeparatorMenuItem(),
+				itemReportIssue, itemGoogleIssues, new SeparatorMenuItem(), itemAboutPLP,
+				itemSWLicense);
+				
+		menuBar.getMenus().addAll(file, edit, view, project, tools, simulation, help);
+		
+		return Components.wrap(menuBar);
+	}
```

Comment:
This method is incredibly long. This was mentioned before, but we should refactor this in the future. (I have some ideas of how we can clean up this whole class - let's discuss them after this sprint)